Samples/ExplicitlyGroupedKernels/Program.cs

```diff
@@ -146,108 +146,100 @@ class Program
+            using var context = Context.CreateDefault();
+            // For each available device...
+            foreach (var device in context)
             {
-                // For each available device...
-                foreach (var device in context)
-                {
-                    // Create accelerator for the given device
-                    using (var accelerator = device.CreateAccelerator(context))
-                    {
-                        Console.WriteLine($"Performing operations on {accelerator}");
+                // Create accelerator for the given device
+                using var accelerator = device.CreateAccelerator(context);
+                Console.WriteLine($"Performing operations on {accelerator}");
 
-                        var data = Enumerable.Range(1, 128).ToArray();
+                var data = Enumerable.Range(1, 128).ToArray();
 
-                        int groupSize = accelerator.MaxNumThreadsPerGroup;
-                        KernelConfig launchDimension = (
-                            (data.Length + groupSize - 1) / groupSize,  // Compute the number of groups (round up)
-                            groupSize);                                 // Use the given group size
+                int groupSize = accelerator.MaxNumThreadsPerGroup;
+                KernelConfig launchDimension = (
+                    (data.Length + groupSize - 1) / groupSize,  // Compute the number of groups (round up)
+                    groupSize);                                 // Use the given group size
 
-                        using (var dataSource = accelerator.Allocate1D<int>(data.Length))
-                        {
-                            // Initialize data source
-                            dataSource.CopyFromCPU(data);
+                // Initialize data source
+                using var dataSource = accelerator.Allocate1D<int>(data.Length);
+                dataSource.CopyFromCPU(data);
 
-                            using (var dataTarget = accelerator.Allocate1D<int>(data.Length))
-                            {
+                using var dataTarget = accelerator.Allocate1D<int>(data.Length);
 
-                                // Launch default grouped kernel
-                                {
-                                    dataTarget.MemSetToZero();
+                // Launch default grouped kernel
+                {
+                    dataTarget.MemSetToZero();
 
-                                    var groupedKernel = accelerator.LoadStreamKernel<ArrayView<int>, int>(GroupedKernel);
-                                    groupedKernel(launchDimension, dataTarget.View, 64);
+                    var groupedKernel = accelerator.LoadStreamKernel<ArrayView<int>, int>(GroupedKernel);
+                    groupedKernel(launchDimension, dataTarget.View, 64);
 
-                                    accelerator.Synchronize();
+                    accelerator.Synchronize();
```

Comment:
nit: should we remove the explicit synchronization here and place a note that the synchronization happens implicitly by a call to `GetAsArray1D()`?